### DISCUSSION TITLE
DNS Support in address line for easy LAN support

### DISCUSSION THREAD

**Discussion by T-Bonhagen** [rustdesk/rustdesk#4381] [Feature Request]
We would like to have FQDN in the address line. 
As a gov. org. with aprox. 50k win installation we would like to use rustdesk. 
We  have a VPN or LAN setup for all clients, so no need for support over unsecure internet.
We like the LAN support with ip addresses, but with so many clients segmented in subdomains we need FQDN support to use rustdesk for 1st level support.

Any chance to have this in official build?

**Comment by SergeyMy** [comment]
and than it is not pleasant unique ID?
it's more secure than a domain name

**Comment by dinger1986** [comment]
> We would like to have FQDN in the address line. As a gov. org. with aprox. 50k win installation we would like to use rustdesk. We have a VPN or LAN setup for all clients, so no need for support over unsecure internet. We like the LAN support with ip addresses, but with so many clients segmented in subdomains we need FQDN support to use rustdesk for 1st level support.
> 
> Any chance to have this in official build?

hello,

Can you please contact RustDesk for this, either on sales@rustdesk.com or support@rustdesk.com.

Thanks

**Comment by SergeyMy** [comment]
RustDesk licensed under GPLv3
you just need to install (or build) https://rustdesk.com/server/
all clients will be managed by a unique ID, inside and under the control of the company!
https://rustdesk.com/docs/de/self-host/
